x-pack/platform/plugins/shared/onechat/server/services/agents/modes/chat/run_chat_agent.ts

```diff
@@ -73,8 +73,11 @@ export const runChatAgent: RunChatAgentFn = async (
     shareReplay()
   );
 
-  events$.subscribe((event) => {
-    events.emit(event);
+  events$.subscribe({
+    next: (event) => events.emit(event),
+    error: () => {
+      // error will be handled by function return, we just need to trap here
+    },
   });
```

Comment:
Thank you rxjs for making error stacktrace absolutely unusable, causing investigating such bugs to take hours instead of minutes.